### DISCUSSION TITLE
Is it possible to use Nuxt Content 2 for create a forum (reddit like) experience?

### DISCUSSION THREAD

**Discussion by AxelBriche** [nuxt/content#1269] [Q&A]
Hello. I want to allow my user to add markdown content (including WMD syntax) from my production website interface (without a code editor, but with an online editor and save the content inside a database), is it possible?

Example of what I try to do :
1) I want create a forum experience (like reddit) and display an online markdown editor to my users.
2) Allow my user to write a post with the editor (in markdown and mdc syntaxe).
3) Save the post inside a sql database.
4) Display the post with Nuxt content 2 and render the content.

A classic forum experience, boosted with by MDC syntaxe.

**Comment by ManasMadrecha** [comment]
Content Module is useful if you want to query your md files using frontmatter's keys or use Vue Components in markdown. 

Don't use Content Module just for MDC syntax. If you only want MDC, then you can use any js-based web markdown editor, any markdown parser (e.g., micromark) and simply use MDC's remark plugin, i.e., https://www.npmjs.com/package/@docus/remark-mdc-edge.
